app/validators/initiative_has_manager_validator.rb

```diff
@@ -0,0 +1,7 @@
+class InitiativeHasManagerValidator < ActiveModel::Validator
+  def validate(initiative)
+    unless initiative.source.is_a? Manager or initiative.helpers.any? ->(h){ h.is_a? Manager }
```

Comment:
NIT: I recommend avoiding `unless` with complex boolean conditionals as this gets out of hand quickly.

Reply:
Will avoid